effectsize.Rmd

```diff
@@ -0,0 +1,99 @@
+# Effect size {#effectsize}
+<div style="text-align:center">
+```{r}
+knitr::include_graphics("figures/effectsize/teaser.png")
```

Comment:
TODO for @chatchavan: generate this in code. Possibly also update to match the new simple effect size exemplar?

I am also happy to take a stab at converting the existing figure to ggplot code if desired.